docker-java/pom.xml

```diff
@@ -125,16 +125,16 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<!-- Force the version to 2.18.3 to ensure the compatibility with old projects -->
-			<version>2.18.3</version>
```

Comment:
We should consider using a BOM to align these dependencies. See https://github.com/docker-java/docker-java/pull/2446